documentation/2.0/content/release-notes.md

```diff
@@ -35,3 +35,9 @@ java.lang.IllegalArgumentException: In production mode, it's not allowed to set
 **ISSUE**: For WLS versions prior to 14.1.1, there is a problem setting the `GroupMemberOf` attribute in the `topology/Security/User` folder. The value is not persisted correctly, and the assignment will not be present when the domain is started.
 
 **ACTION**: Contact Oracle Support to obtain the patch for bug number 30319071 for your WebLogic Server version before running the tool.
+
+#### Problems setting `RotateLogOnStartup` attribute
+
+**ISSUE**: For WLS versions up to and including 14.1.2, there is a problem setting the `RotateLogOnStartup` attribute in various log file folders. The value is not persisted correctly, and the assignment will not be present when the domain is started.
```

Comment:
WLS 14.1.2 ? -> As far as I know, this release doesn't exist yet. Am I mistaken? If not, then you should not mention it in the docs unless you expect WLS 14.1.2 to go out before WDT 2.0.

Reply:
Changed this to "For existing WLS versions"